### PR TITLE
Gui fixes

### PR DESCRIPTION
**Pull request by blazeuser**
Checkstyle fixes, deletion of App and Controller and change to pom.xml to allow report generating in maven.

### CODE REVIEW

File: src/main/java/screens/GraphScene.java [hunk 1]
```diff
@@ -19,32 +19,59 @@
...
 
         if (event.getSource() instanceof DrawNode) {
+            /**
+             * DrawNode object that is linked to the Node object.
+             */
             DrawNode rect = (DrawNode) (event.getSource());
             state.handle(rect);
         } else if (event.getSource() instanceof Line) {
+            /**
+             * Line object.
```
Comment (tvheems) [inline src/main/java/screens/GraphScene.java]: Why do you have these random comments in the code?

File: src/main/java/screens/LoadingScreen.java [hunk 2]
```diff
@@ -21,6 +28,11 @@ public LoadingScreen() {
         setScene(scene);
     }
 
+    /**
+     * Sets the number to display on the progress bar.
+     *
```
Comment (tvheems) [inline src/main/java/screens/LoadingScreen.java]: Why is this extra line here?

File: src/main/java/screens/GraphScene.java (hunk 1 above)
Comment (blazeuser) [inline src/main/java/screens/GraphScene.java]: Because checkstyle sees this as an error.

Comment (blazeuser) [inline src/main/java/screens/GraphScene.java]: A new object/class (event) is being defined here with new variables not defined earlier. This is probably the reason checkstyle returns an error.

File: src/main/java/screens/InfoScreen.java [hunk 3]
```diff
@@ -10,12 +10,15 @@
  */
 /*package*/ class InfoScreen extends Stage {
```
Comment (DexterQvL) [inline src/main/java/screens/InfoScreen.java]: Another commented modifier

File: src/main/java/screens/InfoScreen.java [hunk 4]
```diff
@@ -10,12 +10,15 @@
...
+    /**
+     * Text area for printing information.
+     */
     private final TextArea textArea;
 
     /**
      * Constructor.
      */
-    public InfoScreen() {
+    /*package*/ InfoScreen() {
```
Comment (DexterQvL) [inline src/main/java/screens/InfoScreen.java]: Another commented modifier

File: src/main/java/screens/NodeCenter.java [hunk 5]
```diff
@@ -12,14 +12,23 @@
...
+     * Screen that displays the graph.
+     */
+    private GraphScene graphScene;
 
+    /**
+     * Constructor.
+     * @param sc Scene for displaying graphs.
+     */
     public NodeCenter(GraphScene sc) {
         this.graphScene = sc;
```
Comment (DexterQvL) [inline src/main/java/screens/NodeCenter.java]: Redundant empty line

File: src/main/java/screens/NodeCenter.java [hunk 6]
```diff
@@ -12,14 +12,23 @@
...
 
     }
 
+
```
Comment (DexterQvL) [inline src/main/java/screens/NodeCenter.java]: Redundant empty line, and this method has no javadoc, I understand that it implements the handle method from the INodeHandler interface, however, the javadoc in that interface also doesn't properly describe what this implementation of the method does.

File: src/main/java/screens/NodeInfo.java [hunk 7]
```diff
@@ -7,12 +7,20 @@
...
 
+    /**
+     * NodeInfo state handler constructor.
+     * @param sc Scene where the graph is drawn.
+     */
     public NodeInfo(GraphScene sc) {
         this.scene = sc;
     }
 
+    @Override
```
Comment (DexterQvL) [inline src/main/java/screens/NodeInfo.java]: Missing proper javadoc (same issue as in NodeCenter).

File: src/main/java/screens/Window.java [hunk 8]
```diff
@@ -106,20 +124,27 @@ public static Backlog getBackLog() {
...
     public static LoadingScreen getLoadingScreen() {
         return loadingScreen;
     }
 
     /**
      * Creates the menu bar with its items.
-     *
      * @param stage Main stage.
      * @return The menu bar object.
      */
```
Comment (DexterQvL) [inline src/main/java/screens/Window.java]: This method is rather long, is there any way you can split this into multiple smaller methods?

File: src/main/java/screens/GraphScene.java [hunk 9]
```diff
@@ -19,32 +19,59 @@
...
+    /**
+     * The current state for node events.
+     */
     private INodeHandler state;
-     /**
+
+    /**
      * Event handler for when a node or edge is clicked.
      */
      private EventHandler<MouseEvent> click = event -> {
```
Comment (blazeuser) [inline src/main/java/screens/GraphScene.java]: Click is not a method. @Override will just result in an error.

File: src/main/java/screens/GraphScene.java [hunk 10]
```diff
@@ -19,32 +19,59 @@
...
-            System.out.println(edgeNodes);
             Window.getInfoScreen().getTextArea().appendText("Edge from node " + edgeNodes.substring(0, edgeNodes.indexOf("-")) + " to " + edgeNodes.substring(edgeNodes.indexOf("-") + 1, edgeNodes.length()) + "\n");
         }
      };
 
-
+    /**
+     * GraphScene pane constructor.
+     */
      /*package*/ GraphScene() {
```
Comment (blazeuser) [inline src/main/java/screens/GraphScene.java]: This modifier is not ignored by the compiler. This prevents this method from being accessed by classes outside this package.

**Pull request merged**